status/templates/status/status.html

```diff
@@ -12,9 +15,37 @@
   <br>
   <!--title-->
   <div class="well">
-    <h2>Status</h2>
+    <h2
+      {% if running_contests %}
+        title='<ol>{% for contest in running_contests %}<li>{{contest}}</li> {% endfor %}</ol>'
+      {% endif %}>
```

Comment:
I put it in the title so that we can see the running contest info as we hover above h2.